redfish-core/lib/hypervisor_system.hpp

```diff
@@ -1416,6 +1439,31 @@ inline void requestRoutesHypervisorSystems(App& app)
                 handleHypSLAACAutoConfigPatch(
                     ifaceId, ethData, *ipv6AutoConfigEnabled, asyncResp);
             }
+
+            if (ipv6StaticDefaultGateways)
+            {
+                const std::vector<std::string>& ipv6StaticDefaultGw = *ipv6StaticDefaultGateways;
+                if (ipv6StaticDefaultGw.size() != 1)
+                {
+                    std::string defaultGwStr = "{\"IPv6StaticDefaultGateways\":[\"";
+                    for (uint i = 0; i < ipv6StaticDefaultGw.size(); i++)
+                    {
+                        defaultGwStr += "\"" + ipv6StaticDefaultGw[i] + "\"";
+                        if (i != ipv6StaticDefaultGw.size() - 1)
+                        {
+                            defaultGwStr += ",";
+                            continue;
+                        }
+                        defaultGwStr += "]}";
+                    }
+                    messages::propertyValueFormatError(
```

Comment:
say if user tries to configure multiple ipv6 default gateways, its correct as per schema,
so returning propertyValueFormatError does not Look right.
thinking if we can do other work around like take only first or last input entry or return a error description to say single entry supported

Reply:
>  its correct as per schema,
Done